cobalt/app/cobalt.cc

```diff
@@ -151,6 +152,14 @@ void SbEventHandle(const SbEvent* event) {
       g_exit_manager = nullptr;
       break;
     }
+    case kSbEventTypeAccessibilityTextToSpeechSettingsChanged: {
```

Comment:
Let's move this out of the middle of the lifecycle events, i.e. put it by kSbEventTypeLink